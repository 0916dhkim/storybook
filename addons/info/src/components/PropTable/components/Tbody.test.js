import React from 'react';
import { shallow } from 'enzyme';

import Tbody from './Tbody';

describe('PropTable/Tbody', () => {
  describe('renders a tbody html node with children', () => {
    const wrapper = shallow(
      <Tbody>
        <div>foo bar</div>
      </Tbody>
    );
    expect(wrapper).toMatchSnapshot();
  });

  describe('renders a tbody html node with multiple children elements', () => {
    const wrapper = shallow(
      <Tbody>
        <div>foo bar</div>
        <div>baz</div>
      </Tbody>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
