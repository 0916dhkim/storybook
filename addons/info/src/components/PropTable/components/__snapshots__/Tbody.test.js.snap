// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<tbody>
  <div>
    foo bar
  </div>
</tbody>
`;

exports[` 2`] = `
<tbody>
  <div>
    foo bar
  </div>
  <div>
    baz
  </div>
</tbody>
`;
